service/src/main/java/com/commercetools/pspadapter/payone/notification/NotificationProcessorBase.java

```diff
@@ -144,6 +151,10 @@ public final void processTransactionStatusNotification(final Notification notifi
      */
     protected abstract boolean canProcess(Notification notification);
 
+    protected boolean isNotCompletedTransaction(@Nonnull Transaction transaction) {
+        return transactionStateResolver.isNotCompletedTransaction(transaction);
+    }
+
     /**
```

Comment:
I think this method "isNotCompletedTransaction" is not need

Reply:
why do you think so? it is used in child classes

Reply:
ok